src/jit/CMakeLists.txt

```diff
@@ -225,6 +222,10 @@ if ((CLR_CMAKE_PLATFORM_ARCH_I386 OR CLR_CMAKE_PLATFORM_ARCH_AMD64) AND WIN32)
     # On x86, build RyuJIT/ARM32 cross-compiling altjit.
     # On amd64, build RyuJIT/ARM64 cross-compiling altjit.
     add_subdirectory(protononjit)
+
+    # Build Linux/x86-running-on-Windows altjit or
+    # Linux/AMD64-running-on-Windows altjit..
+    add_subdirectory(linuxnonjit)
```

Comment:
> What would the benefit of this be? Just so we build fewer things? Who would use it? Developers or CI?

Me. :) I'd love to avoid building the cross-targeting JITs on Windows since I regularly work and build on most of our target platforms and can get the coverage I miss from CI. The Windows build already feels a mite sluggish compared to other platforms; adding another JIT that I'm not likely to use just makes this worse.

> Personally, I don't like building extra stuff. However, it's also gives you a build check that you probably aren't breaking other platform builds.

I don't mind depending on CI for this. It's definitely nice to have the abilities that you're adding, though, so they should be enabled by default.